packages/api/src/generated/schema.graphql

```diff
@@ -905,6 +913,7 @@ type Highlight {
   quote: String
   reactions: [Reaction!]!
   replies: [HighlightReply!]!
+  representation: RepresentationType!
```

Comment:
Can we make it so it defaults to content and isn't required? I think that would make it easier for all the clients

Reply:
Correct me if I'm wrong but I think clients will only fail if is is a required argument/parameter in the graphql request This is a required field in the response which I think should be fine

Reply:
You can test it locally

Reply:
Oh got it, thank you!